src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,111 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private int capasitu = 16;
+    private float loadFactor = 0.75f;
+    private int size;
+    private Node<K,V>[] table;
+
+    private class Node<K,V> {
+        private final K key;
+        private V value;
+        private Node<K,V> next;
+
+        Node(K key, V value,Node<K,V> next) {
+            this.key = key;
```

Comment:
Avoid using getters and setters in the Node class. We have access to the private fields of the inner class, so using getters and setters is redundant.